spec/controllers/searchgov_urls_controller_spec.rb

```diff
@@ -1,26 +1,27 @@
 require 'spec_helper'
 
 describe Admin::SearchgovUrlsController do
-  # let!(:searchgov_domain) { SearchgovDomain.create(domain: 'agency.gov')}
+  fixtures :users
+
+  let(:admin) { users(:affiliate_admin) }
   let(:searchgov_url) { SearchgovUrl.create(url: 'agency.gov/test') }
   let(:params) do
     { association: 'searchgov_urls', parent_scaffold: 'admin/searchgov_domains',
     searchgov_domain_id: searchgov_url.searchgov_domain.id, id: searchgov_url.id }
   end
 
-  describe '#fetch' do
-    # before do
-    #   # post :fetch, association: 'searchgov_urls', parent_scaffold: 'admin/searchgov_domains',
-    #   # searchgov_domain_id: searchgov_url.searchgov_domain.id, id: searchgov_url.id
-    #   # post "/admin/searchgov_domains/#{searchgov_url.searchgov_domain.id}/searchgov_urls/#{searchgov_url.id}/fetch?_method=post&association=searchgov_urls&parent_scaffold=admin%2Fsearchgov_domains&searchgov_domain_id=#{searchgov_url.searchgov_domain.id}"
-    #   post :fetch, params
-    #   p response.body
-    # end
+  context 'when logged in as an admin' do
```

Comment:
@ethanewing , you should be able to use the shared `'super admin logged in'` context for this:
https://github.com/GSA/search-gov/blob/master/spec/support/shared_sites_controller_shared_behavior.rb#L46
https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-context